<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>appointment_dialog</class>
    <widget class="QDialog" name="appointment_dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>420</width>
                <height>420</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Edit appointment</string>
        </property>
        <layout class="QGridLayout" name="grid_layout">
            <item row="0" column="0">
                <layout class="QVBoxLayout" name="vertical_layout">
                    <item>
                        <widget class="QLabel" name="title_label">
                            <property name="font">
                                <font>
                                    <pointsize>18</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Appointment</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignmentFlag::AlignCenter</set>
                            </property>
                            <property name="role" stdset="0">
                                <string notr="true">heading</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="top_layout">
                            <item>
                                <widget class="QLabel" name="operator_label">
                                    <property name="text">
                                        <string>Operator*</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="operators_ckb">
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="cabin_label">
                                    <property name="text">
                                        <string>Cabin*</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="cabins_ckb">
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="time_layout">
                            <item>
                                <widget class="QLabel" name="start_label">
                                    <property name="text">
                                        <string>Start*</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTimeEdit" name="start_tedit">
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="duration_label">
                                    <property name="text">
                                        <string>Duration*</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QSpinBox" name="duration_spinb">
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                    <property name="suffix">
                                        <string>min</string>
                                    </property>
                                    <property name="minimum">
                                        <number>30</number>
                                    </property>
                                    <property name="maximum">
                                        <number>900</number>
                                    </property>
                                    <property name="singleStep">
                                        <number>1</number>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="client_layout">
                            <item>
                                <widget class="QLabel" name="client_label">
                                    <property name="text">
                                        <string>Client*</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="clients_ckb">
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                    <property name="editable">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="treatment_layout">
                            <item>
                                <widget class="QLabel" name="treatment_label">
                                    <property name="text">
                                        <string>Treatment*</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="treatments_ckb">
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                    <property name="editable">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="QLabel" name="notes_label">
                            <property name="text">
                                <string>Notes</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QTextEdit" name="notes_txtedit">
                            <property name="cursor" stdset="0">
                                <cursorShape>IBeamCursor</cursorShape>
                            </property>
                            <property name="tabChangesFocus">
                                <bool>true</bool>
                            </property>
                            <property name="placeholderText">
                                <string>Optional details...</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="buttons_layout">
                            <item>
                                <widget class="QPushButton" name="delete_pbtn">
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                    <property name="toolTip">
                                        <string>Delete this appointment</string>
                                    </property>
                                    <property name="text">
                                        <string>Delete</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="horizontal_spacer">
                                    <property name="orientation">
                                        <enum>Qt::Orientation::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>0</width>
                                            <height>0</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QPushButton" name="cancel_pbtn">
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                    <property name="text">
                                        <string>Cancel</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="save_pbtn">
                                    <property name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                    <property name="text">
                                        <string>Save</string>
                                    </property>
                                    <property name="default">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
